src/ml/neural_net/style_transfer/mps_style_transfer.h

```diff
@@ -0,0 +1,34 @@
+/* Copyright © 2018 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#import <Accelerate/Accelerate.h>
+#import <Foundation/Foundation.h>
+#import <Metal/Metal.h>
+#import <MetalPerformanceShaders/MetalPerformanceShaders.h> 
+
+#import <ml/neural_net/mps_descriptor_utils.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+API_AVAILABLE(macos(10.15))
+@interface TCMPSStyleTransfer : NSObject
+
+@property (nonatomic) NSUInteger batchSize;
+@property (nonatomic) NSNumber *contentLossMultiplier;
+@property (nonatomic) NSNumber *styleLossMultiplier;
+@property (nonatomic) BOOL finetuneAllParams;
```

Comment:
Nit: how about `updateAllParams`?

Reply:
`finetuneAllParams` is `updateAllParams `. It's the name of the property we use  in Python.

<img width="527" alt="Screen Shot 2019-07-25 at 12 54 28 PM" src="https://user-images.githubusercontent.com/4616962/61904304-66f65000-aedb-11e9-84c4-5a5204f7fa1e.png">

Reply:
Eh, I prefer `updateAllParams` in a vacuum, but I see the value in preserving consistency with the Python code, while it still exists...

Reply:
Switched to update all params. The python side can adapt later.